File: Sharebook/Client/src/app/core/Core/Models/TwitterEntities/ExtendedEntities/VideoEntityVariants.ts
import {IVideoEntityVariant} from "../../../../Public/Models/Entities/ExtendedEntities/IVideoEntityVariant";

export class VideoEntityVariant implements IVideoEntityVariant {
  // [JsonProperty("bitrate")]
  public bitrate: number;
  // [JsonProperty("content_type")]
  public contentType: string;
  // [JsonProperty("url")]
  public URL: string;
}

File: Sharebook/Server/src/Services/Post/Sharebook.Post.Domain/AggregatesModel/PostAggregate/TwitterEntities/ExtendedEntities/VideoEntityVariants.cs
namespace Sharebook.Post.Domain.AggregatesModel.PostAggregate.TwitterEntities.ExtendedEntities
{
    using Newtonsoft.Json;
    using Sharebook.Storage.Domain.AggregatesModel.PostAggregate.Entities.ExtendedEntities;

    public class VideoEntityVariant : IVideoEntityVariant
    {
        [JsonProperty("bitrate")]
        public int Bitrate { get; set; }
        [JsonProperty("content_type")]
        public string ContentType { get; set; }
        [JsonProperty("url")]
        public string URL { get; set; }
    }
}